### DISCUSSION TITLE
Help: An unexpected error occured while executing the command...

### DISCUSSION THREAD

**Discussion by z4okolatka** [microsoft/winget-cli#4258] [Q&A]
So, i tried to install firefoxpwa using winget, but got this error
```
An unexpected error occurred while executing the command:
0x80070490 : ������� �� ������.
```
I searched some info about this and found nothing. Tried `winget upgrade` and got same error
Most likely it's because recently i tried to move AppData from C to D, so is there a way to reinstall or completely reset winget?

**Comment by denelon** [comment]
You could try installing the Microsoft.WinGet.Client PowerShell module and running `Repair-WinGetPackageManager -Force -Latest`. Or you can follow the steps at Microsoft Learn used to install WinGet in the Windows Sandbox. 
>Note: You don't need to launch the sandbox, it's just that the commands there will download the App Installer and it's dependencies.

**Reply by z4okolatka** [reply]
I installed Microsoft.WinGet.Client, now it says "-Force" is invalid argument
When i try without -Force, it's running ok and then logs -1
